vehicle-model-lifecycle/src/generate_model.py

```diff
@@ -97,17 +104,23 @@ def main() -> None:
     """Main entry point for generation of vehicle models."""
     cache_data = get_cache_data()
 
-    if CACHE_KEY not in cache_data:
+    if CACHE_KEY_VSS_FILE not in cache_data:
         return
 
-    model_src_file = cache_data[CACHE_KEY]
+    if CACHE_KEY_UNITS_LIST not in cache_data:
+        return
+
+    model_src_file = cache_data[CACHE_KEY_VSS_FILE]
+    model_unit_file_list = cache_data[CACHE_KEY_UNITS_LIST][1:-1].split(", ")
```

Comment:
to get a list out of the cache. I was not able to write a list to the cache. If this would be possible then yeah. But now I write a string with the list.